tests/integration/parity/common.py

```diff
@@ -117,3 +118,17 @@ def test_personal_sign_and_ecrecover(
             web3,
             unlockable_account,
             unlockable_account_pw)
+
+
+class ParityParityModuleTest(ParityModuleTest):
+    def test_trace_replay_transaction(self, web3, mined_txn_hash):
+        super().test_trace_replay_transaction(web3, mined_txn_hash)
```

Comment:
do we need the method definitions here?  Can't we just replace the whole body of this with a `pass`?  I think pytest still picks up on all of the test function names.

Reply:
This is the method used throughout tests module, I wanted to retain the same scheme in these tests as well.